src/main/resources/stubs/errors/errors.gobra

```diff
@@ -106,17 +53,17 @@ func (e *errorString) Error() string {
     return unfolding e.ErrorMem() in e.s
 }
 
-// ErrUnsupported indicates that a requested operation cannot be performed,
-// because it is unsupported. For example, a call to [os.Link] when using a
-// file system that does not support hard links.
-//
-// Functions and methods should not return this error but should instead
-// return an error including appropriate context that satisfies
-//
-//  errors.Is(err, errors.ErrUnsupported)
-//
-// either by directly wrapping ErrUnsupported or by implementing an [Is] method.
-//
-// Functions and methods should document the cases in which an error
-// wrapping this will be returned.
-var ErrUnsupported = New("unsupported operation")
+var ErrUnsupported = errUnsupported()
```

Comment:
Is it intentional that this package does not specify a duplicable package invariant providing `ErrorMem()` for this "constant"? This would not only show-case the feature but could also useful for verification projects

Reply:
Done